src/openassetio-core/src/managerApi/EntityReferencePagerInterface.cpp

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2013-2023 The Foundry Visionmongers Ltd
+#include <openassetio/managerApi/EntityReferencePagerInterface.hpp>
+
+namespace openassetio {
+inline namespace OPENASSETIO_CORE_ABI_VERSION {
+namespace managerApi {
+
+void EntityReferencePagerInterface::close([[maybe_unused]] const HostSessionPtr& hsp) {}
```

Comment:
[consider][minor] Consistency
```suggestion
void EntityReferencePagerInterface::close([[maybe_unused]] const HostSessionPtr& hostSession) {}
```

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/commit/a3c932e63ee863be8661670c3f4803068972c224